globals.go

```diff
@@ -332,6 +332,10 @@ const (
 	GithubAppToken             = "github_app_token"
 	githubAppPrivateKey        = "github_app_key"
 
+	// GitHub client retry limit and the minimum amount of delay before attempting again.
+	GitHubRetryAttempts = 3
```

Comment:
nit: start the comment with the name of the variable i.e. "GitHubRetryAttempts is ..."